flink-python/pyflink/fn_execution/__init__.py

```diff
@@ -15,3 +15,10 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 ################################################################################
+
+import os
+
+if 'PYFLINK_CYTHON' in os.environ:
```

Comment:
Also update the following documentation: https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/python/environment_variables/?